ansible_runner/utils/capacity.py

```diff
@@ -1,5 +1,10 @@
+import glob
 import multiprocessing
+import os
```

Comment:
You could use `pathlib` since this a Python 3 only project. That would remove the need for some of the other imports since `Path` objects have methods that provide the same functionality.